bazel/foreign_cc/BUILD

```diff
@@ -147,8 +147,8 @@ envoy_cmake_external(
     deps = [
         ":ares",
         ":nghttp2",
-        ":zlib",
         "//external:ssl",
+        "//external:zlib",
```

Comment:
?

Reply:
You added that according to git blame 2020-05-01 22:29:14.

Reply:
I think we can revert this file, it's already taken care by #11022

Reply:
Done.